vignettes/data_cleaning.Rmd

```diff
@@ -56,23 +56,23 @@ pisa$dat[1:5, 1:5]
 
 ## Meta data structure
 
-Meta data are stored in a `GADSdat` object with the following structure: 
+Meta data is stored in a `GADSdat` object with the following structure: 
 
 ```{r meta raw, echo = FALSE}
 extractMeta(gads, vars = c("gender"))
 ```
 
-* `varLabel`: a variable label; describes a variable in the data
-* `format`: the `SPSS` format of a variable; `F` represents numeric, `A` represents character variables
-* `display_width`: the display width of a variable in the `SPSS` format
+* `varLabel`: a variable label; describes a variable in the data.
+* `format`: the `SPSS` format of a variable; `F` represents numeric, `A` represents character variables.
+* `display_width`: the display width of a variable in the `SPSS` format.
```

Comment:
since there is no sentence to finish (these are only bullet points), it is unusual to set a full stop. please leave this as is